src/main/java/seedu/commands/InvalidCommand.java

```diff
@@ -4,8 +4,19 @@
 import seedu.utility.Ui;
 
 public class InvalidCommand extends Command {
+    
+    private String message;
```

Comment:
Dont think command should have this bah should be under the Ui.
So in execute it calls Ui.printerror() and the printerror function prints out the
 "Invalid command. Use \"help\" to show the list of possible commands."